Tests/SonarScanner.MSBuild.PreProcessor.Test/ArgumentProcessorTests.cs

```diff
@@ -719,6 +720,38 @@ public void PreArgProc_UserHome_Set_DirectoryExistsNot_CanNotBeCreated(string pa
             "Provide a valid path for 'sonar.userHome' to a directory that can be created.");
     }
 
+    [DataTestMethod]
+    [DataRow("Test.pfx", "changeit")]
+    [DataRow(@"""Test.p12""", " ")]
```

Comment:
We have problems with argument quoting. e.g. We can not specify an empty argument and surrounding double quotes are not stripped off from the value. See also https://sonarsource.atlassian.net/browse/SCAN4NET-204